src/tweaks/tools.h

```diff
@@ -127,8 +127,14 @@ void tool_generateCueFiles(void *pt)
                      "/mnt/SDCARD/.tmp_update/script/cue_gen.sh");
 }
 
+void tool_buildShortRomGameList(void *pt)
+{
+    _runCommandPopup(tools_short_names[6],
+                     "./bin/gameNameList /mnt/SDCARD");
```

Comment:
A better location could be `/mnt/SDCARD/BIOS/arcade_lists` or something like that

Reply:
are you referring where to put the list or the tool?

Reply:
The list